bundles/org.openhab.binding.hydrawise/README.md

```diff
@@ -63,7 +63,7 @@ Use Cases
 | password           | String  | False    | The Hydrawise account password                                                                                           |
 | savePassword       | Boolean | False    | By default the password will be not be persisted after the first login attempt unless this is true, defaults to false    |
 | refresh            | Integer | False    | Defaults to a 60 second polling rate, more frequent polling may cause the service to deny requests                       |
-| refreshToken       | Boolean | False    | A oAuth refresh token, this will be automatically configured after the first login and updated as the token is refreshed |
+| refreshToken       | Boolean | False    | An oAuth refresh token, this will be automatically configured after the first login and updated as the token is refreshed |
```

Comment:
Can you restore the formatting of the table by adding a space for the other rows?